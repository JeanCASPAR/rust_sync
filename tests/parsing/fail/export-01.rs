use rustre::sync;

sync! {
    #![pass(0)]

    #[export]
    #[export]
    node oui() = ();
}

fn main() {}
