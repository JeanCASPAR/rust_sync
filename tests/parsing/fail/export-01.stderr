error: Multiple #[export] attributes
 --> tests/parsing/fail/export-01.rs:8:5
  |
8 |     node oui() = ();
  |     ^^^^
